topics/lua-api.md

```diff
@@ -79,11 +79,15 @@ The only libraries that Valkey ships with and that you can use in scripts are li
 
 While the sandbox prevents users from declaring globals, the execution context is pre-populated with several of these.
 
-### The _redis_ singleton
+### The _server_ singleton
 
-The _redis_ singleton is an object instance that's accessible from all scripts.
+The _server_ singleton is an object instance that's accessible from all scripts.
 It provides the API to interact with Valkey from scripts.
-Its description follows [below](#redis_object).
+Its description follows [below](#server_object).
+
+**Note:**
+For compatibility with Redis, Valkey also exposes a _redis_ top level object, that exposes the exact same set of APIs as the _server_ object.
+Valkey does not intend to drop compatibility for this _redis_ API, but it is recommended to use the _server_ object for newly developed scripts.
```

Comment:
If we keep the "Since" information, we can explain here that the `server` object exists since 7.2.5. Earlier versions (Redis OSS) only support the `redis` object.